bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx75.html

```diff
@@ -0,0 +1,153 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/whatsnew_75" %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ _('What’s new with Firefox') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_75') }}
+{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-whatsnew75' %}
```

Comment:
Setting these types of variables in templates should no longer be needed, since they are now provided automatically in the view as context variables: https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/views.py#L533-L536